resources/js/Components/Selects/MySimpleSelect.tsx

```diff
@@ -0,0 +1,47 @@
+
+import React,{useMemo,memo} from 'react';
+
+import Select from 'react-select';
+import Wrap from './Wrap';
+import { ISelect } from './Interface';
+import {SelectedOption} from "./index"
+import   "/resources/css/custom-select.css";
+
+const  MySimpleSelect:React.FC<ISelect>=memo(({
+    options,
+    value,
+    setField,
+    className,
+    name,
+    errors,
+    title,
+    srcIcon,
+    isClearable=true
+  })=> {
+    
+    const memoizedValue = useMemo(() => SelectedOption(value,options), [value])  
+
+    return (<Wrap name={name} srcIcon={srcIcon} className={className} title={title} errors={errors}>
+     <Select
+        className="col"
+        classNamePrefix="select"
+        value={memoizedValue}
+        autoFocus={false}
+        isDisabled={false}
+        isLoading={false}
+        isClearable={isClearable}
+        isRtl={false}
+        isSearchable={true}
+        name={name}
+        options={options}
+        onChange={o=>{
```

Comment:
You need to make the component more versatile, so that it doesn't send SetField, but just OnChange
onChange ={OnChange}

Reply:
I don't think so, because inside select is using a structure like options={  value: string, label: string;}  for onchange and selected. Outside this component i get only result string or array of strings

Reply:
I'm going to drop two links below. The first one is to my InputSelect component, and the second one is where I use it in my work. Maybe it's correct, but Nick required me to make the components unique.
You can see how I did it for an example
https://github.com/am0nshi/jobs/blob/main/resources/js/Components/InputSelect/index.tsx
https://github.com/am0nshi/jobs/blob/main/resources/js/Components/Profile/MyProfile/MyProfile.tsx

Reply:
You can safely ask Nick what's best. It's just what he wants me to do.)

Reply:
did it